ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -184,12 +172,14 @@ server {
 
         try_files $uri =404;
```

Comment:
what about this one?

Reply:
I'd say to keep this on for the "latest API" because there's no autoindex here and etc. And this page is kinda a continuation of the website, so having the website 404 kinda makes sense.